redhat-cop/helm-charts
# Add Ansible Automation Platform 2.0 charts

#### What is this PR About?
Adds a subscription to Ansible Automation Platform using the new early access operator and creates an instance

#### How do we test this?
I have included what looks like sane tests with the charts. This should be easy to install in a `crc` instance with the default values.yaml and could add more customization to the chart in the near future as needed.

**_Note_** this was based heavily on the model used for installing ArgoCD.

cc: @redhat-cop/day-in-the-life
